config_panel.toml

```diff
@@ -0,0 +1,198 @@
+        bind = "bridge>username_template:__FINALPATH__/config.yaml"
+
+[encryption]
+name = "Encryption"
+services = ["__APP__"]
+    [encryption.config]
+    name = "Encryption settings"
+
+        [encryption.config.encryption]
+        ask = "Enable encryption"
+        type = "boolean"
+        yes = "true"
+		no = "false"
+        help = "Enables end to bridge encryption"
+        bind = "encryption>allow:__FINALPATH__/config.yaml"
+
+        [encryption.config.default_encryption]
+        ask = "Default encryption"
+        type = "boolean"
+        yes = "true"
+        no = "false"
+        help = "Force-enable encryption in all portals the bridge creates. This will cause the bridge bot to be in private chats for the encryption to work properly."
+        bind = "encryption>default:__FINALPATH__/config.yaml"
+
+        [encryption.config.require_encryption]
+        ask = "Require encryption"
+        type = "boolean"
+        yes = "true"
+        no = "false"
+        help = "Require encryption, drop any unencrypted messages."
+        bind = "encryption>require:__FINALPATH__/config.yaml"
+
+[usermanagement]
+name = "User management"
+services = ["__APP__"]
+
+    [usermanagement.config]
+    name = "User management"
+
+        [usermanagement.config.helptext]
+        ask = "Allowed values:\n&emsp;&emsp;* - All Matrix users\n&emsp;&emsp;domain - All users on that homeserver\n&emsp;&emsp;mxid (@user:matrix.org) - Specific Matrix user\n&emsp;&emsp;username - Specific local user"
+        type = "markdown"
+
+        [usermanagement.config.listuser]
+        ask = "Users"
+        type = "tags"
+        visible = "role == 'user'"
+
+        [usermanagement.config.listadmin]
+        ask = "Admins"
+        type = "tags"
+        visible = "role == 'admin'"
+
+        [usermanagement.config.listrelay]
+        ask = "Relay users"
+        type = "tags"
+        visible = "role == 'relay'"
+
+        [usermanagement.config.role]
+        ask = "Role"
+        type = "select"
+        choices = ["user", "admin", "relay"]
+
+[privacy]
+name = "Privacy"
+services = ["__APP__"]
+
+    [privacy.config]
+    name = "Privacy settings"
+
+        [privacy.config.delivery_receipts]
+        ask = "Delivery receipts"
+        type = "boolean"
+        yes = "true"
+        no = "false"
+        help = "Should the bridge send a read receipt from the bridge bot when a message has been sent to WhatsApp?"
+        bind = "bridge>delivery_receipts:__FINALPATH__/config.yaml"
+
+        [privacy.config.send_presence_on_typing]
+        ask = "Send presence on typing"
+        type = "boolean"
+        yes = "true"
+        no = "false"
+        help = "Send the presence as 'available' to whatsapp when users start typing on a portal."
+        bind = "bridge>send_presence_on_typing:__FINALPATH__/config.yaml"
+
+        [privacy.config.url_previews]
+        ask = "URL previews"
+        type = "boolean"
+        yes = "true"
+        no = "false"
```

Comment:
Minor comment but are those really needed each time ? x_X

Reply:
to be tested, if the bridge reads the config with `yes` instead of `true`

Reply:
I'll test this ASAP and change the implementation if needed :)

Reply:
up @thardev

Reply:
Sorry for the delay, it seems that it needs to be a bool value, I tried using `yes` as value but it did not work

Reply:
to be tested also, what happens if you just remove those lines
```
        yes = "true"
        no = "false"
```